addons/stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -72,11 +72,6 @@ def _compute_allowed_picking_ids(self):
                 ('company_id', '=', batch.company_id.id),
                 ('immediate_transfer', '=', False),
                 ('state', 'in', domain_states),
-                '|',
-                '|',
-                ('batch_id', '=', False),
-                ('batch_id', '=', batch.id),
-                ('batch_id', 'in', cancelled_batch_ids),
```

Comment:
You can also remove `cancelled_batchs`/`cancelled_batch_ids` just a little above.

Reply:
You can also modify the `_sanity_check` message as it's asking the pickings aren't already a part of a batch.